src/content/docs/game-design/godot/dungeoncrawler/1-player/index.mdx

```diff
@@ -0,0 +1,145 @@
+    This is where we'll create our player's animations. Rename the *General* animation to *Idle* and click the *Add frames from File* Button (The folder icon)
+
+3. Navigate to your *assets/frames* folder, and decide which character you want to be your player. I'll be using the Plague Doctor. Using Shift + Click select all the frames for your character labeled *Idle* (This should be four frames) then open them.
+
+    ![Idle Animation Frames](/src/assets/godot/dungeonCrawler/idleanim.png)
+
+    You'll see them added to the animation timeline. 
+
+4. We'll want to select two things in the timeline. The **Loop** Button (To ensure the animation loops) and the **Play on start button** (To ensure the animation plays automatically)
+    Let's also increase the FPS to 8 so that the animation plays a little faster. 
+
+    ![Idle Animation Frames](/src/assets/godot/dungeonCrawler/autoloop.png)
+
+    Hit play to test! You'll see the player now has an idle animation that loops!
+
+5. Let's add our character's walking animation. Add a new animation using the **Add animation** Button and call it something like "Walk"
+
+    ![Idle Animation Frames](/src/assets/godot/dungeonCrawler/addanim.png)
+
+    Do the same thing we did to grab the frames for the idle animation, but this time, grab all the frames labeled "Run" it should again be 4 frames. We want this animation to loop, but we **don't** want it to autoplay. Let's also give this a framerate of 8 FPS.
+
+    ![Walk Animation Frames](/src/assets/godot/dungeonCrawler/walkanim.png)
+
+    Finally, just select your idle animation again, to make sure this is what our player will start on.
+
+    Great! That's our animations all done!
+</Steps>
+
+### Collision
+
+Now that we have something to look at, let's give our player a hitbox. 
+<Steps>
+1. Open the inspector for the **CollisionShape2D** we added, and add a new shape in the **empty** shape field.
+    It's a good idea to use a **CapsuleShape** as it will make us less likely to get stuck on corners. Position and adjust the capsule so that it's *slightly* smaller than the sprite for our player.
+    Mine looks like this:
+
+    ![CollissionShape](/src/assets/godot/dungeonCrawler/collshapeimg.png)
+
+2. Great! Our player now has collision. We'll do one more thing while we're here, which is give our player a script to handle movement. Right click on the **CharacterBody2D** and Attach a script. Call it something like "player.gd" and make sure we're saving it in our Scripts folder.
+ We also need to make sure we **untick** the **template** box as we will not be using the template! This is because the template is designed for gravity based platformers.
+</Steps>
+
+With our script created and attached, let's get to programming our movement!
+
+### Movement
+
+<Steps>
+
+1. First, let's set up a variable to control our speed.
+
+    ```gdscript
+    @export var speed = 200
```

Comment:
Please show this in context of the wider program so they know `extends CHaracterBody2D` is above it etc